### PR TITLE
core/vm: dedup config check in markdown logger

### PR DESCRIPTION
**Pull request by gballet**

### CODE REVIEW

**Review by holiman — APPROVED** [review]
LGTM

File: core/vm/logger.go [hunk 1]
```diff
@@ -322,20 +322,21 @@ func (t *mdLogger) CaptureStart(from common.Address, to common.Address, create b
...
 			a = append(a, fmt.Sprintf("%d", elem))
 		}
 		b := fmt.Sprintf("[%v]", strings.Join(a, ","))
 		fmt.Fprintf(t.out, "%10v |", b)
-	}
-	if !t.cfg.DisableStack { // format return stack
-		var a []string
+
+		// format return stack
+		a = nil
```
Comment (karalabe) [inline core/vm/logger.go:335]: Nitpick, `a = a[:0]` is the canonical way to truncate a slice back down for reuse.

Comment (karalabe) [reply core/vm/logger.go:335]: The benefit is that you won't allocate items to grow it back, rather it will just reuse the previous buffers.

Comment (gballet) [reply core/vm/logger.go:335]: yeah that makes sense :+1:

**Pull request merged**